src/storable.rs

```diff
@@ -21,6 +21,10 @@ pub trait BoundedStorable: Storable {
     fn max_size() -> u32;
 }
 
+pub trait EncodableStorable: BoundedStorable {
+    const FIXED_LEN: bool;
```

Comment:
I'd put this as part of the BoundedStorable trait. No need for a separate trait imo.

Reply:
Done.